src/honeysql_postgres/format.clj

```diff
@@ -42,21 +43,88 @@
 (doseq [[k v] postgres-clause-priorities]
   (register-clause! k v))
 
-(defn get-first [x]
+(defn- get-first
+  "Returns the first element if the passed argument is a collection, else return the passed argument
+   as is."
+  [x]
   (if (coll? x)
     (first x)
     x))
 
+(defn- comma-join-args
+  "Returns the args comma-joined after applying to-sql to them"
+  [args]
+  (if (nil? args)
+    ""
+    (->> args
+         (map to-sql)
+         comma-join
+         paren-wrap)))
+
+;; TODO : Add create-table, drop-table and its clauses to the priority clause map
```

Comment:
Did you miss this Todo? Don't you require these entries in the map now that you have the functionality implemented?